### DISCUSSION TITLE
media_content_id changing?

### DISCUSSION THREAD

**Discussion by sni66274** [music-assistant/support#1003] [Q&A] [answered]
I have some automations running which select the media content id like this:
media_content_id: database://radio/41
however with every restart of HA I have to renumber all the buttons with a higher number, is there another way of doing this or can I set a fixed number somewhere?

**Comment by OzGav** [comment, marked as answer]
Have you looked at the FAQ for how to use `mass.queue_command`

**Reply by sni66274** [reply]
No, I didn't but you are right, this works, thx!
